adoc/bp_chap_suma_backup.adoc

```diff
@@ -64,55 +64,59 @@ Running out of space can result in backups failing, so check this often.
 == Backing up {productname}
 
 
-The most comprehensive and simplest method to back up your {productname} installation is to back up the entire [path]``/var/spacewalk/`` directory.
+The most comprehensive method for backing up your {productname} installation is to back up the relevant files and directories.
 This can save you time in administering your backup, and can be faster to reinstall and re-synchronize in the case of failure.
-However, this method requires additional disk space and will take longer to perform the backup.
-The [path]``/var/spacewalk/packages/NULL/`` directory is primarily a duplicate of the package repository.
-In the case of failure, re-synchronize your installation with the [command]``mgr-sync`` tool.
+However, this method requires significant disk space and could take a long time to perform the backup.
+
+
+If you want to only back up the required files and directories, use this list:
 
 [NOTE]
 ====
-If you are running {productname} in a disconnected setup [path]``/var/spacewalk/``__must__ be backed up.
+To make this process simpler, and more comprehensive, we recommend backing up the entire [path]``/etc`` and [path]``/root`` directories, not just the ones specified here.
 ====
 
-If you want to only back up the required files and directories, use this list:
 
-* [path]``/var/spacewalk/``
-* [path]``/etc/sysconfig/rhn/``
+* [path]``/etc/cobbler/``
+* [path]``/etc/dhcp.conf``
+* [path]``/etc/fstab`` and any ISO mountpoints you require.
 * [path]``/etc/rhn/``
+* [path]``/etc/salt``
 * [path]``/etc/sudoers``
-* [path]``/srv/www/htdocs/pub/``
-* [path]``/var/spacewalk/packages/``
-** Each item within this directory represents an organization using custom RPMs, ensure you back up each of them
+* [path]``/etc/sysconfig/rhn/``
 * [path]``/root/.gnupg/``
+* [path]``/root/.ssh``
+** This file exists if you are using an SSH tunnel or SSH [command]``push``. You will also need to have saved a copy of the ``id-susemanager`` key.
 * [path]``/root/ssl-build/``
-* [path]``/etc/dhcp.conf``
+* [path]``/srv/salt``
 * [path]``/srv/tftpboot/``
-* [path]``/etc/cobbler/``
+* [path]``/srv/www/cobbler``
+* [path]``/srv/www/htdocs/pub/``
 * [path]``/var/lib/cobbler/``
 * [path]``/var/lib/rhn/kickstarts/``
-* [path]``/srv/www/cobbler``
-* [path]``/root/.ssh``
-** This file exists if you are using an SSH tunnel or SSH [command]``push``. You will also need to have saved a copy of the ``id-susemanager`` key.
+* [path]``/var/spacewalk/``
+* [path]``/var/spacewalk/packages/``
```

Comment:
Do we need to name `packages` if we already mirror full /var/spacewalk ?

Reply:
That's a good point, probably not.